iOSClient/AppDelegate.swift

```diff
@@ -708,9 +739,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         }
                 
         if account == "" { return }
-        guard let passcode = CCUtility.getPasscode() else { return }
-        if passcode.count == 0 || CCUtility.getNotPasscodeAtStart() { return }
+        if !CCUtility.isPasscodeEnableAtStart() { return }
                 
+        // Dismiss present window?.rootViewController? [ONLY PASSCODE]
+        let presentedViewController = window?.rootViewController?.presentedViewController
+        if presentedViewController is NCLoginNavigationController {
```

Comment:
What does this do?
Shouldn't it check if it is a `TOPasscodeViewController` (to avoid multiple passcode views)?

Reply:
If exists a precedence window?.rootViewController?.present(..) the passcodeViewController don't works, so before, I dismiss the presentedViewController (for example an open in..) except that NCLoginNavigationController (login) because  NCLoginNavigationController wins on Passcode